src/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,73 @@
+// REACT
+import * as React from 'react';
+// STORYBOOK
+import { storiesOf } from '@storybook/react';
+import { select, boolean, text } from '@storybook/addon-knobs';
+// VENDOR
+import styled, { ThemeProvider } from '@xstyled/styled-components';
+// COMPONENTS
+import { Tooltip } from './Tooltip.component';
+import { Button } from '../Button';
+import { Typography } from '../Typography';
+// README
+import * as README from './README.md';
+// THEME
+import { RootTheme } from '../theme';
+
+const StyledStory = styled('div')`
+    padding: 2rem 5rem;
+`;
+
+storiesOf('Components/Tooltip', module)
+    .addParameters({
+        readme: {
+            sidebar: README,
+        },
+    })
+    .add('Default', () => (
+        <StyledStory>
+            <ThemeProvider theme={RootTheme}>
+                <>
+                    <Typography tag="h1">Top Right</Typography>
```

Comment:
Perhaps a more general heading, since this changes with the knobs

Reply:
👍